library/telegraf

```diff
@@ -1,7 +1,7 @@
 Maintainers: David Reimschussel <dreimschussel@influxdata.com> (@reimda),
              Steven Soroka <ssoroka@influxdata.com> (@ssoroka)
 GitRepo: git://github.com/influxdata/influxdata-docker
-GitCommit: 52a89544a93622f32e4967758e678d83f9c691f9
+GitCommit: 281f367f5f46b84e290478e34a8a402541e08af9
```

Comment:
This commit doesn't seem to exist on `influxdata/influxdata-docker`, perhaps you meant https://github.com/influxdata/influxdata-docker/commit/7622ac6ed5040ba1c9e014d2169e34faeb2d7f57